code/modules/mob/living/carbon/human/human.dm

```diff
@@ -358,6 +358,14 @@
 			var/obj/item/place_item = usr.get_active_hand() // Item to place in the pocket, if it's empty
 
 			var/delay_denominator = 1
+			var/pickpocket = 0
+			if(ishuman(usr))
+				var/mob/living/carbon/human/H = usr
+				if(H.gloves && istype(H.gloves,/obj/item/clothing/gloves))
+					var/obj/item/clothing/gloves/G = H.gloves
+					pickpocket = G.pickpocket
+			if(pickpocket)
```

Comment:
i'd say use the pickpocket var to decide the delay,so we can have middle way gloves that can be crafted and help with pickpocketing?